hack/lib/lint.sh

```diff
@@ -61,7 +61,7 @@ kubeedge::lint::check() {
     fi
 
     [[ $(git diff --name-only) ]] && {
-      echo "Some files have white noise issue, please run \`make lint\` to slove this issue"
+      echo "Some files have white noise issue, please run \`git status\` to find whitenoise and solve this issue"
```

Comment:
```suggestion
      echo "Some files have white noise issue, please run \`make lint\` to solve and \`git status\` to find and fix this issue"
```

Reply:
If users find this issue in the ci, they also need to run make lint in local env.

Reply:
ok